core/admin/mailu/sso/views/base.py

```diff
@@ -40,7 +40,10 @@ def login():
             flask_login.login_user(user)
             response = flask.redirect(destination)
             response.set_cookie('rate_limit', utils.limiter.device_cookie(username), max_age=31536000, path=flask.url_for('sso.login'), secure=app.config['SESSION_COOKIE_SECURE'], httponly=True)
-            flask.current_app.logger.info(f'Login succeeded for {username} from {client_ip}.')
+            flask.current_app.logger.info(f'Login succeeded for {username} from {client_ip} pwned={form.pwned.data}.')
+            breaches = int(form.pwned.data)
```

Comment:
Does this need a try/except?
I think we should add it to not spam the logs if the field is empty or filled with garbage.

```python
            try:
                breaches = int(form.pwned.data)
            except ValueError:
                breaches = -1
```

Reply:
Done in 66250e39